src/main.rs

```diff
@@ -63,1116 +66,895 @@ fn main() -> Result<()> {
-                )
-                .arg(verbose_arg.clone())
-        )
-        .subcommand(
-            SubCommand::with_name("login")
-                .about(&*format!("{} Authenticate Wrangler with your Cloudflare username and password", emoji::UNLOCKED)))
-        .subcommand(
-            SubCommand::with_name("report")
-                .about(&*format!("{} Report an error caught by wrangler to Cloudflare", emoji::BUG))
-                .arg(
-                    Arg::with_name("log")
-                        .long("log")
-                        .takes_value(true)
-                        .help("specifies a log to report (e.g. --log=1619728882567.log)")
-                )
-        )
-        .get_matches();
-
-    let mut is_preview = false;
-
-    let not_recommended_msg = styles::warning("(Not Recommended)");
-    let recommended_cmd_msg = styles::highlight("`wrangler config --api-key`");
-    let wrangler_login_msg = styles::highlight("`wrangler login`");
-    let api_token_url = styles::url("https://dash.cloudflare.com/profile/api-tokens");
-    let token_support_url = styles::url(
-        "https://support.cloudflare.com/hc/en-us/articles/200167836-Managing-API-Tokens-and-Keys",
-    );
-
-    if let Some(matches) = matches.subcommand_matches("config") {
-        // If api-key flag isn't present, use the default auth option (API token)
-        let default = !matches.is_present("api-key");
-
-        let user: GlobalUser = if default {
-            // API Tokens are the default
-            StdOut::billboard(&format!("To find your API Token, go to {}\nand create it using the \"Edit Cloudflare Workers\" template.\n\nConsider using {} which only requires your Cloudflare username and password.\n\nIf you are trying to use your Global API Key instead of an API Token\n{}, run {}.", api_token_url, wrangler_login_msg, not_recommended_msg, recommended_cmd_msg));
-            let api_token: String = interactive::get_user_input("Enter API Token: ");
-            GlobalUser::TokenAuth { api_token }
-        } else {
-            StdOut::billboard(&format!("We don't recommend using your Global API Key!\nPlease consider using an API Token instead.\n\n{}", token_support_url));
-            let email: String = interactive::get_user_input("Enter Email: ");
-            let api_key: String = interactive::get_user_input("Enter Global API Key: ");
+#[derive(Debug, StructOpt)]
+#[structopt(
+    name = "wrangler",
+    author = "The Wrangler Team <wrangler@cloudflare.com>",
+    setting = AppSettings::ArgRequiredElseHelp,
+    setting = AppSettings::DeriveDisplayOrder,
+    setting = AppSettings::VersionlessSubcommands,
+)]
+pub struct Cli {
+    /// Toggle verbose output (when applicable)
+    #[structopt(name = "verbose", long = "verbose", global = true)]
+    pub verbose: bool,
+
+    /// Path to configuration file.
+    #[structopt(
+        name = "config",
+        short = "c",
+        long = "config",
+        default_value = "./wrangler.toml",
```

Comment:
Could you change "./wrangler.toml" to "wrangler.toml" in the `default_value` attr key? I can't push to your fork.